tests/dataframe_tests/test_dataframe_aggregation.py

```diff
@@ -87,8 +90,33 @@ def test_translate_count_aggregation(translator):
     )
     result_df = translator.aggregate_nested_properties()
 
-    print(result_df.show(truncate=False))
     assert_dataframe_equality(expected_df, result_df, get_node_id_name("imaging_study"))
     diff = []
     assert_zero(result_df, diff, ["_dx_series_file_count", "_mr_series_file_count"])
     assert diff == [], f"Differences: {diff}"
+
+@pytest.mark.schema_parent
+@pytest.mark.parametrize("translator", [("parent", "participant", "aggregation", [])], indirect=True)
+def test_flatten_nested_array_parent_props(translator):
+    """
+    Test to ensure the created dataframe will not contains any array being nested in another array
+    - input dataframe with leaf nodes containing nested list
+    - expected dataframe with leaf nodes no longer containing nested list
+    :param translator:
+    :return:
+    """
+    print("Start parent testing")
+    [input_df, expected_df] = get_dataframes_from_names(
+        get_spark_session(translator.sc),
+        "parent",
+        ["participant__0_Translator.translate_table_to_dataframe__participant",
+        "participant__0_Translator.translate_parent"]
+    )
+    result_df = translator.translate_parent(input_df)
+    print(result_df.show(truncate=False))
+    field = result_df.schema["data_type"]
+    assert isinstance(field.dataType, ArrayType)
+    filter_df = result_df.filter(result_df._participant_id == "80cc940b-414f-4361-ac9f-24a94279e379").select("data_type")
+    data_types = filter_df.first()["data_type"]
+    print(data_types)
+    assert all_match(data_types, ["csv", "tsv", "json"]) is True
```

Comment:
what would have been the result before this fix instead of `["csv", "tsv", "json"]`?

Reply:
it is [["csv", "json"], ["tsv", "json"]p